Apps/APN-UIKit/APN UIKit/AppDelegate.swift

```diff
@@ -51,6 +51,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
         // Add event listeners for in-app. This is not to initialise in-app but event listeners for in-app.
         MessagingInApp.initialize(eventListener: self)
+        MessagingPushAPN.configure { config in
```

Comment:
can we maybe combine `configure` and `initialize`? maybe, initializing returns `config` that can be used?

Reply:
To +1 to Shahroz's comment, the SDK does have a naming pattern that we could duplicate in the Push module, too. 

Since we already use `initialize` in other places: 

`CustomerIO.initialize()`
`MessagingInApp.initialize()`

It would be good if the push module was also `initialize`.

Reply:
I named the method `configure` because initialization is not a mandatory step in this case. The main purpose of this API is to configure the `MessagingPushAPN` module. However, since two votes are in favor of `initialize`, I will change the name to `initialize`. Also, note that we do not currently have an initialization method in `MessagingPushAPN`.

Reply:
✅